client/post-editor/editor-drawer/label.jsx

```diff
@@ -3,16 +3,25 @@
  */
 import React, { PropTypes } from 'react';
 
-export default function EditorDrawerLabel( { children } ) {
+/**
+ * Internal dependencies
+ */
+import InfoPopover from 'components/info-popover';
+
+export default function EditorDrawerLabel( { children, labelText, helpText } ) {
 	return (
 		<label className="editor-drawer__label">
 			<span className="editor-drawer__label-text">
+				{ labelText }
+				{ helpText && <InfoPopover position="top left">{ helpText }</InfoPopover> }
 				{ children }
```

Comment:
> then I see no reason we need a separate labelText

The reasoning for still needing this prop, after moving { children } out of the span is info popover positioning gets _interesting_ then.